pipes/WDL/tasks/tasks_intrahost.wdl

```diff
@@ -83,6 +87,8 @@ task polyphonia_detect_cross_contamination {
       ~{'--min-readcount ' + min_readcount} \
       ~{'--max-mismatches ' + max_mismatches} \
       ~{'--min-maf ' + min_maf} \
+      ~{'--masked-positions ' + masked_positions} \
+      --masked-positions-file ~{masked_positions_file} \
```

Comment:
If masked positions file is optional, you should use the same optional parameter syntax as masked positions so it works properly if empty

Reply:
Updated, thank you!